.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: erlang
 otp_release:
+  - 18.2.1
   - 18.0
-  - 17.4
-  - R16B
```

Comment:
Dropping Erlang < 0.18 is fine on master: see #1139.

However, is the plan to merge this into 0.x as well? I would prefer so. Does rebar3 require Erlang 18?